### ISSUE TITLE
[BUG] on PERMISSIONS_POLICY regex

### ISSUE THREAD

**Issue by thelittlefireman** [bunkerity/bunkerweb#532]
**Description**
Concise description of what you're trying to do, the expected behavior and the current bug.
`sync-xhr=(self \"https://api.pwnedpasswords.com\" \"https://2fa.directory\")` is not handle by the PERMISSIONS_POLICY regex

**How to reproduce**
`      - bitwarden.test.test_PERMISSIONS_POLICY=accelerometer=(), ambient-light-sensor=(), autoplay=(), battery=(), camera=(), cross-origin-isolated=(), display-capture=(), document-domain=(), encrypted-media=(), execution-while-not-rendered=(), execution-while-out-of-viewport=(), fullscreen=(), geolocation=(), gyroscope=(), hid=(), idle-detection=(), magnetometer=(), microphone=(), midi=(), navigation-override=(), payment=(), picture-in-picture=(), publickey-credentials-get=(self), screen-wake-lock=(), serial=(), usb=(), web-share=(), xr-spatial-tracking=(), sync-xhr=(self \"https://api.pwnedpasswords.com\" \"https://2fa.directory\")
`

**Logs**
```
[2023-06-25 22:08:41] - GENERATOR - ⚠️  - Ignoring variable XXXXX_PERMISSIONS_POLICY : value accelerometer=(), ambient-light-sensor=(), autoplay=(), battery=(), camera=(), cross-origin-isolated=(), display-capture=(), document-domain=(), encrypted-media=(), execution-while-not-rendered=(), execution-while-out-of-viewport=(), fullscreen=(), geolocation=(), gyroscope=(), hid=(), idle-detection=(), magnetometer=(), microphone=(), midi=(), navigation-override=(), payment=(), picture-in-picture=(), publickey-credentials-get=(self), screen-wake-lock=(), serial=(), usb=(), web-share=(), xr-spatial-tracking=(), sync-xhr=(self \"https://api.pwnedpasswords.com\" \"https://2fa.directory\") doesn't match regex ^(?![, ])(,? ?([a-z-]+)(?!.*[^-]\2=)=(\*|\(( ?(self|\u0022https?:\/\/[-\w@:%.+~#=]+[-\w()!@:%+.~#?&\/=$]*\u0022))*\)))*$
[
```

**Comment by fl0ppy-d1sk** [comment]
Should be fixed in 1.5.1 since we now use pure LUA to add headers.

**Comment by fl0ppy-d1sk** [comment]
Hello @thelittlefireman,

Backslashes are not needed anymore in the last 1.5.1 release. 🥳
